packages/mdc-select/component.ts

```diff
@@ -180,11 +173,11 @@ export class MDCSelect extends MDCComponent<MDCSelectFoundation> implements MDCR
       }
     };
 
-    this.targetElement_.addEventListener('change', this.handleChange_);
-    this.targetElement_.addEventListener('focus', this.handleFocus_);
-    this.targetElement_.addEventListener('blur', this.handleBlur_);
+    this.selectedText_.addEventListener('change', this.handleChange_);
+    this.selectedText_.addEventListener('focus', this.handleFocus_);
+    this.selectedText_.addEventListener('blur', this.handleBlur_);
 
-    this.targetElement_.addEventListener('click', this.handleClick_ as EventListener);
+    this.selectedText_.addEventListener('click', this.handleClick_ as EventListener);
 
     if (this.menuElement_) {
```

Comment:
We might not need to check for this condition since menu will always be present.